src/backend/run_tests.py

```diff
@@ -0,0 +1,16 @@
+from subprocess import Popen
+
+directories = ["./blockchain", 
+                "./services/wallet-api", 
+                "./services/transaction-api", 
+                "./services/users-api", 
+                "./services/mining-api"]
+
+change_directory = ["cd"]
+run_coverage_pytest = ["python", "-m", "coverage", "run", "--source=./src", "-m", "pytest"]
```

Comment:
Looks great, only comment I have is should this be python3 instead of python? And should we then be using pip3 over pip?

Reply:
good question, I guess that's device-dependent. python should recognize python3 if that's the default python version. I will change it to python3 just to be safe.

Reply:
Perfect, approved then!

Reply:
So this is weird but python3 fails to find some dependencies on my device. Can you confirm if current scripts work for you? If they do I would prefer if we stick with python

Reply:
You’ll have to install the dependencies with pip3 instead of pip

Reply:
I’m just commenting from my phone rn but I can run them in the morning

Reply:
I tried that, doesn't work for dotenv for some reason. I will confirm in a few minutes

Reply:
So I tried troubleshooting the issue, no luck. I tried deleting my venv and made sure to use pip3 and python3 to run the program. The file doesn't work with the python3 subprocess even when running the file with python3.